Configs/backtranslate/bt_baseline.yaml

```diff
@@ -0,0 +1,18 @@
+defaults:
+  - data: en-de
+  - train_eval: train_eval_baseline
```

Comment:
Small thing, if not too much work: all the values for `train_eval` that are `train_eval_X` could just be `X`, e.g. `- train_eval: baseline`

Reply:
not at all, changed